packages/webapp/public/locales/en/translation.json

```diff
@@ -604,93 +551,56 @@
       "UPLOAD_LINK_MESSAGE": "Download this template",
       "UPLOAD_PLACEHOLDER": "Upload CSV file",
       "VALIDATION": {
-        "EXTERNAL_ID": "Invalid external id, must be between 1 and 40 characters.",
         "FILE_ROW_LIMIT_EXCEEDED": "File row limit exceeded. The max number of sensors that can be uploaded from a file is 100.",
-        "MISSING_COLUMNS": "Columns are required/missing.",
-        "SENSOR_BRAND": "Invalid brand name, must be between 1 and 100 characters.",
-        "SENSOR_DEPTH": "Invalid depth value, must be between 0 and 1000.",
-        "SENSOR_HARDWARE_VERSION": "Invalid hardware version name, must be between 1 and 100 characters.",
         "SENSOR_LATITUDE": "Invalid latitude value, must be between -85 and 85. and fewer than 10 decimals.",
         "SENSOR_LONGITUDE": "Invalid longitude value, must be between -180 and 180. and fewer than 10 decimals.",
-        "SENSOR_MODEL": "Invalid model name, must be between 1 and 100 characters.",
-        "SENSOR_NAME": "Invalid sensor name, must be between 1 and 100 characters.",
-        "SENSOR_READING_TYPES": "Invalid reading type detected: {{ reading_types }}. Valid values include: soil_water_content, soil_water_potential, temperature."
+        "SENSOR_NAME": "Invalid sensor name, must be between 1 and 100 characters."
       }
```

Comment:
Are they used in the shared folder? If so, I might want to take some time on reviewing this PR later...
https://github.com/LiteFarmOrg/LiteFarm/blob/95aa1465a0dd3a2ac75324bc2aac3f9381689d2b/packages/shared/validation/sensorCSV.js#L19-L34

Reply:
UGH OH NO! The shared folder!

Reply:
Probably should fix this in the future. i18n script should be able to recognize them.
I am not sure why translations keys can't be used directly.
If translation keys are calculated t(map[propFromDb]), those translation keys usually goes to a separate translation file